app/controllers/InitialiseAppealController.scala

```diff
@@ -60,5 +75,7 @@ class InitialiseAppealController @Inject()(appealService: AppealService,
       .addingToSession((SessionKeys.endDateOfPeriod, appealModel.endDate.toString))
       .addingToSession((SessionKeys.dueDateOfPeriod, appealModel.dueDate.toString))
       .addingToSession((SessionKeys.dateCommunicationSent, appealModel.dateCommunicationSent.toString))
+      .addingToSession((SessionKeys.reasonableExcuse, if (isAppealAgainstObligation) "obligation" else ""))
```

Comment:
We have a `isObligationAppeal` session key, and there is some other code that relies on that session key

Reply:
Doesn't matter on the value of `isObligationAppeal` just as long as the key is present

Reply:
Ok missed that will update

Reply:
Updated based on feedback